api/src/main/java/org/apache/unomi/api/services/ProfileService.java

```diff
@@ -110,6 +110,10 @@ public interface ProfileService {
 
     void addAliasToProfile(String profileID, String alias, String clientID);
 
+    ProfileAlias removeAliasFromProfile(String profileID, String alias, String clientID);
```

Comment:
Could you please add Javadoc for all the profile alias methods ? This is a requirement of the API classes

Reply:
done